Changes.md

```diff
@@ -21,7 +21,7 @@
 * TLS: don't abort loading certs in config/tls dir when an error is encountered.
   Process every cert file and then emit errors. #2729
 * fix connection pool not being unique when hosts and ports were equal between domains #2788
-
+* smtp_forward: this plugin does not use the queue_outbound hook anymore
```

Comment:
can you specify in this PR when it stopped using that hook?

Reply:
Will do

Reply:
Done